example/grid_example.py

```diff
@@ -141,7 +141,9 @@ def grid_mutil_yaxis() -> Grid:
     )
 
     bar.overlap(line)
-    return Grid().add(bar, opts.GridOpts(pos_left="5%", pos_right="20%"))
+    return Grid().add(
+        bar, opts.GridOpts(pos_left="5%", pos_right="20%"), is_control_axis_index=True
```

Comment:
In general, opts.GridOpts, and other Opts are design leaks. Why leak? opts.* are all developers' abstractions to make things logical and coherent. The leak is that we exposed our abstraction to end user. This is a thought coming to my mind when pyecharts 0.5.x users complains about the 1.x's usability.